tests/suites/test_suite_shax.data

```diff
@@ -1,3 +1,9 @@
+SHA-1 - Valid parameters
+sha1_valid_param:
```

Comment:
Do we need `depends_on:MBEDTLS_SHA1_C` for these tests? A similar question for the 256 and 512 cases as well.

Reply:
Yes, I've just added it.